firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/NewReleaseFetcherTest.java

```diff
@@ -286,6 +287,31 @@ public void handleNewReleaseFromClient_whenNewAabIsAvailable_returnsRelease() th
         result);
   }
 
+  @Test
+  public void
+      handleNewReleaseFromClient_whenNewReleaseHasDifferentVersionNameThanInstalled_returnsRelease()
+          throws Exception {
+    when(mockFirebaseAppDistributionTesterApiClient.fetchNewRelease(
+            any(), any(), any(), any(), any()))
+        .thenReturn(
+            getTestNewReleaseDifferentVersionName()
```

Comment:
nit: extract this to a variable so you don't need to set the builder fields again below

Reply:
Good call. Changed the other methods to also have this change.